rust/lance-encoding/src/encodings/physical/byte_stream_split.rs

```diff
@@ -257,6 +262,55 @@ impl MiniBlockDecompressor for ByteStreamSplitDecompressor {
     }
 }
 
+/// Determine if BSS should be used based on mode and data characteristics
+pub fn should_use_bss(data: &FixedWidthDataBlock, mode: BssMode) -> bool {
+    // Only support byte-aligned data
+    let bytes_per_value = data.bits_per_value / 8;
+    if data.bits_per_value % 8 != 0 || bytes_per_value < 2 {
+        return false;
+    }
```

Comment:
I thought somewhere else you limited it to 32 bits or 64 bits but here you are allowing 16 bits so that seems inconsistent?

Reply:
You are right, let me fix this,